operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -477,8 +488,19 @@ def addClusteredServer(self, cluster, server):
     self.writeln("- name: " + name)
     self.writeln("  listenPort: " + str(server.getListenPort()))
     self.writeln("  listenAddress: " + self.quote(self.env.toDNS1123Legal(self.env.getDomainUID() + "-" + server.getName())))
+    self.writeln("  adminPort: " + str(server.getAdministrationPort()))
```

Comment:
perhaps just don't include the adminPort in the output if adminPort is disabled?   ditto for SSL, or the default channel.     (Even the default channel can be disabled.)